easybuild/easyconfigs/m/meshio/meshio-5.3.5-foss-2023b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'meshio'
+version = '5.3.5'
+
+homepage = 'https://github.com/nschloe/meshio'
+description = "meshio is a tool for reading/writing various mesh formats representing unstructured meshes"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),  # includes numpy
```

Comment:
This comment is actually not true anymore for recent Python versions, numpy is now part of SciPy-bundle. The other dependencies listed at https://github.com/nschloe/meshio/blob/v5.3.5/pyproject.toml#L40 are both part of Python-bundle-PyPI. Both bundles are dependencies of VTK, which is why this is probably installing without issues, but we should list them here explicitly.

Reply:
```suggestion
    ('Python', '3.11.5'),
    ('Python-bundle-PyPI', '2023.10'),  # includes rich
    ('SciPy-bundle', '2023.11'),  # includes numpy
```